internal/gtkcord/sidebar/guilds/folder.go

```diff
@@ -158,6 +157,22 @@ func (f *Folder) Set(folder *gateway.GuildFolder) {
 	for _, g := range f.Guilds {
 		g.Invalidate()
 	}
+
+	// After guilds are loaded, read their labels and set the folder name if unset.
+	if len(folder.Name) == 0 {
+		for i, g := range f.Guilds {
```

Comment:
It'd be better if you do

```go
var name string
for ... {
    name += "..."
}
f.Name.SetName(name)
```

Modifying `*gateway.GuildFolder` may not be the best idea.

Reply:
okay will do, sorry am a bit new to go, didn't know funcs were pass by reference